### PR TITLE
Added DOCS: Interact with DSA on smart contract level

### PR DESCRIPTION
**Pull request by bhavik-m**
Added DOCS: Interact with DSA on smart contract level

### CODE REVIEW

File: content/en/get-started/cast (smart contract).md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+        // encoding data to run multiple things through cast on account
+        // Depositing in DSA and then deposit in Compound through DSA.
+        string[] memory _targets = new string[](2);
+        bytes[] memory _data = new bytes[](2);
+        
+        _targets[0] = "BASIC-A";
+        _targets[1] = "COMPOUND-A";
+        
+        bytes4 private constant basicDeposit = bytes4(keccak256("deposit(address,uint256,uint256,uint256)"));
+        bytes4 private constant compoundDeposit = bytes4(keccak256("deposit(string,uint256,uint256,uint256)"));
```
Comment (thrilok209) [inline content/en/get-started/cast (smart contract).md:50]:
```suggestion
        bytes4 memory basicDeposit = bytes4(keccak256("deposit(address,uint256,uint256,uint256)"));
        bytes4 memory compoundDeposit = bytes4(keccak256("deposit(string,uint256,uint256,uint256)"));
```

File: content/en/get-started/cast (smart contract).md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+        
+        address dai = 0x6B175474E89094C44Da98b954EedeAC495271d0F;
+        uint amtToDeposit = 1e18; // 1 DAI
+        
+        _data[0] = abi.encodeWithSelector(basicDeposit, dai, amtToDeposit, 0, 0);
+        _data[1] = abi.encodeWithSelector(compoundDeposit, dai, amtToDeposit, 0, 0);
```
Comment (thrilok209) [inline content/en/get-started/cast (smart contract).md:56]:
```suggestion
        _data[1] = abi.encodeWithSelector(compoundDeposit, "DAI-A", amtToDeposit, 0, 0);
```

File: content/en/get-started/cast (smart contract).md [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+```
+owner: Owner of the Smart Account
+accountVersion: Account Module version
+_origin: Where Smart Account is created.
+```
+
+* [cast(_targets, _datas, _origin):](https://github.com/Instadapp/dsa-contracts/blob/master/contracts/v2/accounts/module1/Implementation_m1.sol#L80)
+Using cast() user can access the connectors, which allows the smart account to interact with protocols or set up any settings on the smart account.
+
+```
```
Comment (pradyuman-verma) [inline content/en/get-started/cast (smart contract).md:89]: Better to put this parameter in Table format.
like this :
image

Comment (bhavik-m) [reply content/en/get-started/cast (smart contract).md:89]: Done

**Review by KaymasJain — APPROVED** [review]

**Pull request merged**